README.md

```diff
@@ -94,6 +94,8 @@ You also have to provide the following environment variables from an account tha
 - `RESINTEST_REGISTER_PASSWORD`: The password of the account to register.
 - `RESINTEST_REGISTER_USERNAME`: The username of the account to register.
```

Comment:
Actually, since the recurly token is used by the API's env vars configuration, the best thing that I can thing is to use the new `RESINTEST_API_URL` var this this PR also adds and target the api server that you prefer. Does this sound good enough?